tests/core/tests.rs

```diff
@@ -3,10 +3,13 @@ mod tests {
     use super::*;
 
     fn deser_test<T: Deserialize + ToBytes>(orig: &T) {
-        print_cbor_types("orig", &orig.to_bytes());
-        let deser = T::deserialize(&mut Deserializer::from(std::io::Cursor::new(orig.to_bytes()))).unwrap();
+        let orig_bytes = orig.to_bytes();
+        print_cbor_types("orig", &orig_bytes);
+        let mut deserializer = Deserializer::from(std::io::Cursor::new(orig_bytes.clone()));
+        let deser = T::deserialize(&mut deserializer).unwrap();
         print_cbor_types("deser", &deser.to_bytes());
         assert_eq!(orig.to_bytes(), deser.to_bytes());
```

Comment:
This check was never failing in this PR, but I just added it in just to be sure. It can't hurt.